public/assets/gws/script.js

```diff
@@ -4,26 +4,26 @@
s=!0,t.updateOffset(this,!0),this._changeInProgress=null)),this):this._isUTC?n:It(this):null!=e?this:NaN}function Lt(t,e){return null!=t?("string"!=typeof t&&(t=-t),this.utcOffset(t,e),this):-this.utcOffset()}function Ht(t){return this.utcOffset(0,t)}function Nt(t){return this._isUTC&&(this.utcOffset(0,t),this._isUTC=!1,t&&this.subtract(It(this),"m")),this}function Wt(){return this._tzm?this.utcOffset(this._tzm):"string"==typeof this._i&&this.utcOffset(At(ci,this._i)),this}function Bt(t){return this.isValid()?(t=t?Ct(t).utcOffset():0,(this.utcOffset()-t)%60===0):!1}function Gt(){return this.utcOffset()>this.clone().month(0).utcOffset()||this.utcOffset()>this.clone().month(5).utcOffset()}function zt(){if(!c(this._isDSTShifted))return this._isDSTShifted;var t={};if(d(t,this),t=Tt(t),t._a){var e=t._isUTC?a(t._a):Ct(t._a);this._isDSTShifted=this.isValid()&&v(t._a,e.toArray())>0}else this._isDSTShifted=!1;return this._isDSTShifted}function Yt(){return this.isValid()?!this._isUTC:!1}function Ut(){return this.isValid()?this._isUTC:!1}function Vt(){return this.isValid()?this._isUTC&&0===this._offset:!1}function qt(t,e){var o,i,n,s=t,a=null;return jt(t)?s={ms:t._milliseconds,d:t._days,M:t._months}:"number"==typeof t?(s={},e?s[e]=t:s.milliseconds=t):(a=Gi.exec(t))?(o="-"===a[1]?-1:1,s={y:0,d:m(a[wi])*o,h:m(a[Si])*o,m:m(a[xi])*o,s:m(a[Ti])*o,ms:m(a[ki])*o}):(a=zi.exec(t))?(o="-"===a[1]?-1:1,s={y:$t(a[2],o),M:$t(a[3],o),d:$t(a[4],o),h:$t(a[5],o),m:$t(a[6],o),s:$t(a[7],o),w:$t(a[8],o)}):null==s?s={}:"object"==typeof s&&("from"in s||"to"in s)&&(n=Kt(Ct(s.from),Ct(s.to)),s={},s.ms=n.milliseconds,s.M=n.months),i=new Pt(s),jt(t)&&r(t,"_locale")&&(i._locale=t._locale),i}function $t(t,e){var o=t&&parseFloat(t.replace(",","."));return(isNaN(o)?0:o)*e}function Xt(t,e){var o={milliseconds:0,months:0};return o.months=e.month()-t.month()+12*(e.year()-t.year()),t.clone().add(o.months,"M").isAfter(e)&&--o.months,o.milliseconds=+e-+t.clone().add(o.months,"M"),o}function Kt(t,e){var o;return t.isValid()&&e.isValid()?(e=Ft(e,t),t.isBefore(e)?o=Xt(t,e):(o=Xt(e,t),o.milliseconds=-o.milliseconds,o.months=-o.months),o):{milliseconds:0,months:0}}function Jt(t,e){return function(o,i){var n,r;return null===i||isNaN(+i)||(ot(e,"moment()."+e+"(period, number) is deprecated. Please use moment()."+e+"(number, period)."),r=o,o=i,i=r),o="string"==typeof o?+o:o,n=qt(o,i),Zt(this,n,t),this}}function Zt(e,o,i,n){var r=o._milliseconds,s=o._days,a=o._months;e.isValid()&&(n=null==n?!0:n,r&&e._d.setTime(+e._d+r*i),s&&R(e,"Date",j(e,"Date")+s*i),a&&K(e,j(e,"Month")+a*i),n&&t.updateOffset(e,s||a))}function Qt(t,e){var o=t||Ct(),i=Ft(o,this).startOf("day"),n=this.diff(i,"days",!0),r=-6>n?"sameElse":-1>n?"lastWeek":0>n?"lastDay":1>n?"sameDay":2>n?"nextDay":7>n?"nextWeek":"sameElse",s=e&&(_(e[r])?e[r]():e[r]);return this.format(s||this.localeData().calendar(r,this,Ct(o)))}function te(){return new f(this)}function ee(t,e){var o=g(t)?t:Ct(t);return this.isValid()&&o.isValid()?(e=E(c(e)?"millisecond":e
```

Comment:
__[jshint]__
* Use '===' to compare with '0'.
* Expected an assignment or function call and instead saw an expression.
* 'a' is already defined.
* 's' is already defined.
* 'l' is already defined.
* 'u' is already defined.
* Too many errors. (24% scanned).

<!-- Comment by [SideCI](https://sideci.com) -->